src/leo-client-app/src/pages/_app.tsx

```diff
@@ -11,6 +11,10 @@ export const metadata: Metadata = {
   description: "Generated by create next app",
 };
 
+/*
```

Comment:
nit: follow more conventional docstring format:

i.e
/*
* Component is the active page
* UserProvider is a wrapper that allows us to access
* auth0 features globally throughout the app
/*

Reply:
ok the formatting looks weird but just add a * before every line

Reply:
if this is a eslint issue then ignore